bin/setup_cluster.sh

```diff
@@ -95,8 +95,9 @@ function setup_gcloud_credentials() {
 
 # use mason_client to get resources
 function get_resource() {
-  # Turn off modules, otherwise the go.mod file will be updated
-  GO111MODULE=off go get istio.io/test-infra/boskos/cmd/mason_client
+  # cd to tmp, otherwise the go.mod file may be updated
+  # shellcheck disable=SC2164
+  (cd /tmp; go get istio.io/test-infra/boskos/cmd/mason_client)
```

Comment:
does this pr to fix the recent perf pipeline error: 
```
 get_resource gke-perf-preset perf-tests /tmp/vLAfu.boskos.info /tmp/Bxton.boskos.log
+ GO111MODULE=off
+ go get istio.io/test-infra/boskos/cmd/mason_client
# istio.io/test-infra/boskos/cmd/mason_client
../test-infra/boskos/cmd/mason_client/main.go:141:46: too many arguments in call to mason.NewClient
	have (*client.Client, error)
	want (mason.boskosClient)
../test-infra/boskos/cmd/mason_client/main.go:141:63: not enough arguments in call to client.NewClient
	have (string, string)
	want (string, string, string, string)
+ cleanup
+ [[ True == \T\r\u\e ]]
```

Reply:
yes

Reply:
also this PR fixes periodic upgrade tests error: 
```
+ get_resource gke-e2e-test upgrade-tests /tmp/E9uhK.boskos.info /tmp/O31nu.boskos.log
+ GO111MODULE=off
+ go get istio.io/test-infra/boskos/cmd/mason_client
# istio.io/test-infra/toolbox/util
../test-infra/toolbox/util/githubClient.go:274:9: cannot use "created" (type string) as type *string in field value
+ cleanup
+ [[ True == \T\r\u\e ]]
```

Reply:
@howardjohn thanks!